__tests__/intergration.test.js

```diff
@@ -256,33 +256,73 @@ describe("POST: /api/articles/:article_id/comments", () => {
       .send(malformedBody)
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("Bad request");
+        expect(body.msg).toBe("Bad request - malformed body");
       });
   });
-  test("400: sends an appropriate message and status when passed a non-valid user", () => {
+  test("400: sends an appropriate message and status when passed a invalid id format", () => {
+    const malformedBody = {
+      username: "butter_bridge",
+      body: "great comment",
+    };
+    return request(app)
+      .post("/api/articles/not-a-number/comments")
+      .send(malformedBody)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request - malformed body");
```

Comment:
...and just be careful with the message coming back, whilst this is a bad request, the body is fine for this test.